README.md

```diff
@@ -55,7 +63,9 @@ To easily differentiate PRs against snapshot, staging and production, each PR to
 
 # Package promotion
 
-A package can go through different stages from snapshot to staging to production. To promote a package from one distribution to another, it must first be added to the new distribution and as soon as it is added, removed from the old distribution. As an example, a package `foo-1.2.3` is in `snapshot`. Now the content is copied over to `staging` and as soon as the package is merged, the package `foo-1.2.3` should be removed from `snapshot`. If the same package version exists in both branches, the first one is taken. In the above case this is `staging`.
+A package will usually go through different stages from snapshot to staging to production.  It is recommended to use the [elastic/elastic-package](https://github.com/elastic/elastic-package) repo to achieve package promotion, and it is the developer's burden to fulfill tests on snapshot stage and to coordinate any desired tests on staging prior to promoting to production storage.  Note that production is in use by all 7.10+ released versions of Elastic and testing should reflect this to avoid problems. 
```

Comment:
```suggestion
A package will usually go through different stages from snapshot to staging to production.  It is recommended to use the [`elastic-package` tool](https://github.com/elastic/elastic-package) to achieve package promotion. It is the package developer's burden to fulfill tests on snapshot stage and to coordinate any desired tests on staging prior to promoting to production storage.  Note that production is in use by all 7.10+ released versions of Elastic and testing should reflect this to avoid problems. 
```